cardano/src/block/normal.rs

```diff
@@ -104,8 +104,9 @@ impl cbor_event::de::Deserialize for DlgProof {
 
 #[derive(Debug, Clone)]
 pub struct TxPayload {
-    txaux: Vec<tx::TxAux>,
+    pub txaux: Vec<tx::TxAux>,
```

Comment:
I'd rather not expose internal members. In fact, I don't understand the purpose of this type which is just a wrapper over `Vec<tx::TxAux>`. Perhaps more principled accessor methods are needed on `Tx`, such as:

```rust
    fn contents(&self) -> &[tx::TxAux] {
        // ...
    }
```